site/assets/scripts/app.js

```diff
@@ -91,17 +100,39 @@ $(document).ready(function() {
       promise.done(function(){
         showLayerDetails(App.dataSet.catalogKey);
       });
+      datasourceChanged = false;
     }
   }); 
+
+  // Added by Louis - allows for switching between Census and Synthea data catalogs
+  $("#data_source_switch").change(function(e) {
+    const datasource = $(e.target).val();
+    if (datasource === "census")
+    {
```

Comment:
align with the format of the rest of the file { on the same line for an if, and clean up the extra white space